storage/users.go

```diff
@@ -76,11 +85,66 @@ type Storage struct {
 	DB *sqlx.DB
 }
 
-func (s *Storage) getUser(ctx context.Context, col string, val interface{}) (*User, error) {
-	var u userModel
+const getUserQuery = `
+	SELECT
+	  users.*,
+	  m.membership,
+	  ips.ip_whitelist
+	FROM
+	  users
+	  LEFT JOIN (
+		SELECT
+		  membership.user_id,
+		  json_agg(
+			json_build_object(
+			  'tenant_id',
+			  membership.tenant_id,
+			  'type',
```

Comment:
The column `membership.tenant_type` does not exist. Removing this will allow logins to work.

But `tenant_type` json prop is mapped to `membership.membership_type` which looks wrong. I'm not following exactly what's going on here. @e-asphyx @carte7000 can one of you take a closer look please.

Reply:
I think @carte7000's idea was to expose tenant.type too but he'd made a mistake and I CPed it everywhere :)

Reply:
You are right @e-asphyx , I have a fix on my local service account branch, will push it soon I just need to fix integration test and this should be good

Reply:
Fixed